tests/queries/0_stateless/01254_dict_load_after_detach_attach.sql

```diff
@@ -17,7 +17,7 @@ LAYOUT(FLAT());
 DETACH DATABASE dict_db_01254;
 ATTACH DATABASE dict_db_01254;
 
-SELECT query_count, status FROM system.dictionaries WHERE database = 'dict_db_01254' AND name = 'dict';
```

Comment:
Hm, I see that the ATTACH query loaded the dictionary asynchronously, but is this correct?

Reply:
Not sure, I'd like to see a better investigation.